### DISCUSSION TITLE
By design or bug?

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [skeletonlabs/skeleton#1099] [Q&A] [answered]
I noticed that the Lightswitch changed between versions 0.99.5 and 0.132.5 (Not sure when the change actually happens)

I find it weird on the newer version, the pill background has almost no contrast at all and it's hard to distinguish.

Also noticed that the color of the trigger itself changed but this goes more in taste I guess. Same with the position, dark used to be on the right side, now it's on the left side.

![TOGGLE](https://user-images.githubusercontent.com/8037559/223181673-fa7604e0-9411-48a7-a015-499f9a780790.png)

**Comment by riziles** [comment]
I don't want to speak for them, but I believe the developers completely overhauled it in order to make it much easier to customize. You should be able to specify your own background color now with the `bgLight` and `bgDark` props. You can see all the changes here:

https://github.com/skeletonlabs/skeleton/commit/855318f004d365f42009bfe7619d368764313981

and here:

https://github.com/skeletonlabs/skeleton/commit/71070ee15f20a81937a3b6230a84176be6f83a9e

**Comment by endigo9740** [comment, marked as answer]
@MrAmericanMike the Lightswitch now adapts to your theme colors. Does your background behind the lightswitch not change when you toggle light/dark mode?

Here's what I see on the preset Skeleton theme when overlapping a card element for example:

![Screenshot 2023-03-06 at 5 37 44 PM](https://user-images.githubusercontent.com/1509726/223281330-55b2caf2-d8fb-4170-b209-a8e2a2605e5e.png)

![Screenshot 2023-03-06 at 5 37 57 PM](https://user-images.githubusercontent.com/1509726/223281332-bb755d1b-801c-49fd-b75d-85b38e8fc317.png)

**Reply by MrAmericanMike** [reply]
> @MrAmericanMike the Lightswitch now adapts to your theme colors. Does your background behind the lightswitch not change when you toggle light/dark mode?
> 
> Here's what I see on the preset Skeleton theme when overlapping a card element for example:
> 
> ![Screenshot 2023-03-06 at 5 37 44 PM](https://user-images.githubusercontent.com/1509726/223281330-55b2caf2-d8fb-4170-b209-a8e2a2605e5e.png)
> 
> ![Screenshot 2023-03-06 at 5 37 57 PM](https://user-images.githubusercontent.com/1509726/223281332-bb755d1b-801c-49fd-b75d-85b38e8fc317.png)

Thanks. Yes it does changes it background, I guess the less contrast is a result of now using the Crimson Theme colors, but as mentioned now it's more customizable, so I guess I can change that to give it more contrast. I though maybe it was some big or missing styles.

Thanks for the response.
